renku/command/schema/workflow_file.py

```diff
@@ -33,6 +33,23 @@ class Meta:
         model = WorkflowFilePlan
         unknown = marshmallow.EXCLUDE
 
+    @marshmallow.pre_dump(pass_many=True)
+    def fix_ids(self, objs, many, **kwargs):
```

Comment:
Should we do this for Plan's inputs, outputs, and parameters as well?

Reply:
I didn't see any input/output/param ids that had this issue, but I can add it just to be safe